Cython/Compiler/PyrexTypes.py

```diff
@@ -3107,6 +3131,10 @@ def __init__(self, return_type, args, has_varargs = 0,
         self.args = args
         self.has_varargs = has_varargs
         self.optional_arg_count = optional_arg_count
+        if isinstance(exception_value, (int, float, str, bytes)):
```

Comment:
Alternative 2:
```suggestion
        if not isinstance(exception_value, self.ExceptionValue):
            assert isinstance(exception_value, (int, float, str, bytes))
```

Reply:
Again, this is what I'd have liked to do, but it can also be a `Node`. I could check for `Node` too here, but I was starting to worry about circular imports